app/src/main/res/layout/listitem_user_equipment_set_new.xml

```diff
@@ -0,0 +1,40 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content">
+
+    <androidx.appcompat.widget.AppCompatTextView
+        android:id="@+id/new_equipment_set_label"
+        style="@style/TextHeadlineHigh"
+        android:layout_width="0dp"
+        android:layout_height="wrap_content"
+        android:layout_marginStart="@dimen/margin_medium"
+        android:layout_marginTop="@dimen/margin_medium"
+        android:layout_marginBottom="@dimen/margin_medium"
+        android:gravity="center_vertical"
+        android:maxLines="1"
+        android:text="@string/user_equipment_set_new_set"
+        app:autoSizeMinTextSize="14dp"
```

Comment:
Prefer a standard text size dimen.

Reply:
Our app uses the same 14dp everywhere :(. There is no standard text size dimen